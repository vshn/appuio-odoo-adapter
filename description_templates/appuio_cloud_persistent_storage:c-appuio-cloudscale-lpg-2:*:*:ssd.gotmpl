{{- $access := "RWO" -}}
{{- $class  := "ssd" -}}

{{- template "_storage" dict "AccessMode" $access "StorageClassName" $class "Item" . -}}
