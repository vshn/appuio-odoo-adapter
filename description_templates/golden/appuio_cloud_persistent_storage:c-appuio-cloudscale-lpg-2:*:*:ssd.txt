Persistent Storage

Storage Type: RWO [ssd]
Qty: 87955674.09 UNIT-Minutes
Average Usage: 7.61 UNIT / Minute
Average Memory requests: 0.70 TPS / Minute
Average CPU requests in memory request equivalent: 0.70 TPS / Minute
Unit Price: CHF 0.0000007460 / UNIT / Minute